docs/docs/ja/index.md

```diff
@@ -1,24 +1,24 @@
 ---
 title: Tuistとは？
-description: Apple の公式ツールを拡張し、大規模なアプリをより優れたものにします。
+description: Apple のネイティブツールを拡張し、スケールするアプリをより良くする。
 ---
 
-<h1 id="tuist">Tuistについて</h1>
+# Tuistについて {#tuist}
 
 アプリ開発の世界、特に Apple のようなプラットフォームでは、組織はしばしば **生産性の問題** に直面します。これには、遅いコンパイル時間、不確実なテスト、リソースを消耗する複雑な自動化ワークフローが含まれます。 従来、企業は専任のプラットフォームチームを結成してこれらの問題に対処しています。 これらの専門家はコードベースの健全性と整合性を維持し、他の開発者が機能の開発に集中できるようにします。 しかし、このアプローチは高コストでリスクが伴う可能性があり、重要な役割を担うチームメンバーの退職が生産性に深刻な影響を及ぼすことがあります。
 
 **Tuist は、アプリ開発を加速し、強化するために設計されたツールチェーンです。** 私たちは公式ツールやシステムとシームレスに統合し、開発者が馴染みのある環境で作業できるようサポートします。 ツールやシステムの統合の負担を軽減することで、チームが機能開発と全体的な開発者体験の向上にエネルギーを注げるようにします。 要するに、Tuistはあなたのプロジェクトを支えるチームのような役割を果たします。 アプリアイディアの閃きからユーザーへのリリースまで、私たちはあなたと共に歩み、発生する課題に取り組みます。
 
 Tuist は、開発者の主要な入り口である **[CLI](https://github.com/tuist/tuist)** と、CLI が状態を保持し、他の公開サービスと統合するためのサーバーで構成されています。 サーバーを必要とする機能は、サブスクリプションを必要とする場合がありますのでご注意ください。 サーバーを必要とする機能は、サブスクリプションを必要とする場合がありますのでご注意ください。 サーバーを必要とする機能は、サブスクリプションを必要とする場合がありますのでご注意ください。
 
-<h2 id="why-would-i-use-tuist">Tuistを使う理由は？</h2>
+## Tuist を使う理由は？ {#why-would-i-use-tuist}
 
-なぜTuistを選択するのか？ その理由は以下の通りです： その理由は以下の通りです： その理由は以下の通りです：
+なぜTuistを選択するのか？ その理由は以下の通りです。
 
 - **モジュール化を容易にする：** プロジェクトが成長し、複数のプラットフォームにまたがると、モジュール化が重要になります。 Tuistはこの複雑さを簡素化し、プロジェクトの構造を最適化し、よりよく理解するためのツールを提供します。 Tuistはこの複雑さを簡素化し、プロジェクトの構造を最適化し、よりよく理解するためのツールを提供します。
 - **ワークフローの最適化：** プロジェクト情報を活用し、Tuistは選択的なテスト実行とビルド間の決定論的なバイナリ再利用を通じて効率を向上させます。
 - **プロジェクトの健全な成長を促進する：** プロジェクトのダイナミクスに関する洞察と、情報に基づいた意思決定のための専門的なガイダンスを提供します。 このアプローチにより、開発者の離職やビジネスゴールの達成に失敗することに繋がる健全でないプロジェクトによるフラストレーションや生産性の低下を防ぎます。 このアプローチにより、開発者の離職やビジネスゴールの達成に失敗することに繋がる健全でないプロジェクトによるフラストレーションや生産性の低下を防ぎます。
-- **高コストなプラットフォーム専任チームの代替：** 高コストでリスクのある社内の専任チームに投資するのではなく、Tuistをあなたのバーチャル専門家として利用してください。 私たちは、属人化に伴うリスクを避けながら、安定したサポートを提供します。 私たちは、属人化に伴うリスクを避けながら、安定したサポートを提供します。 私たちは、属人化に伴うリスクを避けながら、安定したサポートを提供します。
+- **高コストなプラットフォーム専任チームの代替：** 高コストでリスクのある社内の専任チームに投資するのではなく、Tuistをあなたのバーチャル専門家として利用してください。 私たちは、属人化に伴うリスクを避けながら、安定したサポートを提供します。 私たちは、属人化に伴うリスクを避けながら、安定したサポートを提供します。 私たちは、属人化に伴うリスクを避けながら、安定したサポートを提供します。 私たちは、属人化に伴うリスクを避けながら、安定したサポートを提供します。
```

Comment:
For some reason, the same sentence is being inserted repeatedly. Is it a problem with Crowdin?
Not only in this part, but similar repetitions of the same text can be seen in other places as well.

```suggestion
- **高コストなプラットフォーム専任チームの代替：** 高コストでリスクのある社内の専任チームに投資するのではなく、Tuistをあなたのバーチャル専門家として利用してください。 私たちは、属人化に伴うリスクを避けながら、安定したサポートを提供します。 
```

Reply:
I think this is my fault. Crowdin's system expects modifications to the files in the target language, Japanese in this case, to be done directly through their editor. Otherwise, we can break their segmentation, and they fail at reconciling the changes with their segmentation. Would you mind fixing them directly in the editor?

Apologies for that.